plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSplitManager.java

```diff
@@ -91,20 +92,44 @@ public void testInitialSplits()
         List<DeltaLakeSplit> splits = getSplits(splitManager, deltaLakeConfig);
 
         List<DeltaLakeSplit> expected = ImmutableList.of(
-                makeSplit(0, 5_000, fileSize, minimumAssignedSplitWeight),
-                makeSplit(5_000, 5_000, fileSize, minimumAssignedSplitWeight),
-                makeSplit(10_000, 5_000, fileSize, minimumAssignedSplitWeight),
-                makeSplit(15_000, 5_000, fileSize, minimumAssignedSplitWeight));
+                makeSplit(FULL_PATH, 0, 5_000, fileSize, minimumAssignedSplitWeight),
+                makeSplit(FULL_PATH, 5_000, 5_000, fileSize, minimumAssignedSplitWeight),
+                makeSplit(FULL_PATH, 10_000, 5_000, fileSize, minimumAssignedSplitWeight),
+                makeSplit(FULL_PATH, 15_000, 5_000, fileSize, minimumAssignedSplitWeight));
 
         assertEquals(splits, expected);
     }
 
+    @Test(dataProvider = "absolutePaths")
+    public void testAbsolutePathSplits(final String absoluteRawEncodedFilePath, final String absoluteDecodedParsedFilePath)
```

Comment:
Remove `final`. We don't use it for method arguments.

Reply:
done

Reply:
Reminder.